### ISSUE TITLE
Create new email templates for daily SMS limit warnings/stoppages

### ISSUE THREAD

**Issue by andrewleith** [cds-snc/notification-planning#868]
Create a new email template for when a warning a service they are approaching their limit, for letting them know their service is suspended when they reach their limit, and for when they change their limits via settings.

Note: These templates will need to be "configurable" to say either "email limit" or "sms limit" as appropriate.

## Description

Given a user is close to going over their daily SMS limit, then the service team members will receive a warning email. 

Given a user reaches their daily SMS limit, then the service team members will receive an email that they cannot send anymore SMS that day.

Given a user's daily SMS limit is changed, then the service team members will receive an informational email. 

We will need 3 new templates for this:
1. `Config.NEAR_DAILY_LIMIT_TEMPLATE_ID`: message to be sent when a service is approaching their daily limit. 
1. `Config.REACHED_DAILY_LIMIT_TEMPLATE_ID`: message to be sent when a service hits their daily limit.
1. `DAILY_LIMIT_UPDATED_TEMPLATE_ID`: message to be sent when a client updates their limit settings.  This template needs to be updated to convey both limits (sms + email) to the user whenever either is changed.

- This template will be used when the `FF_SPIKE_SMS_DAILY_LIMIT` feature flag is on.

NOTE: see [this pr](https://github.com/cds-snc/notification-api/commit/614d4e8f49060efcd9b0ecca5229d5ad97237564#diff-713249ee365774535b627819c8f5246ac51f83174c148873c4e8e28d2aebaf4e) for an idea of what needs to be done here

## Acceptance criteria

- [x] Draft templates so users are aware when close to limit and when reach limit
- [x] Send emails for translations
- [x] Translations reviewed
- [x] Complete QA steps
- [x] Review for typos by designers

### Test warnings
- [x] Ensure the new template is received when a service is approaching their daily sms limit (greater than 80%)
- [x] Ensure the new template is received when a service is approaching their daily email limit (greater than 80%) to confirm still working as it should be after we've changed SMS limits

### Test stoppages
- [x] Ensure the new template is received when a service reaches their daily sms limit 
- [x] Ensure the new template is received when a service reaches their daily email limit to confirm still working as it should be after we've changed SMS limits

### Test settings change
- [x] Ensure the new template is received when a service has their daily sms limit changed
- [x] Ensure the new template is received when a service has their daily email limit changed to confirm still working as it should be after we've changed SMS limits

**Comment by yaelberger-commits** [comment]
@andrewleith I think we have the content for this covered in another card. Is this card for the dev implementation of those new templates?

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/868?planning-poker) with Zenhub @andrewleith @jimleroyer @jzbahrai @sastels @smcmurtry

**Comment by yaelberger-commits** [comment]
cc @amazingphilippe and @YedidaZalik that we will need content for each of these 3 new templates. To be covered in #366

**Comment by andrewleith** [comment]
@yaelberger-commits yes this was intended to cover the DEV work to implement the new templates that would cover both sms limits and email limits (vs. the existing ones that just cover one global limit).

**Comment by yaelberger-commits** [comment]
draft PR up, existing content into a Google doc, @YedidaZalik working on content. Need a final decision
text message/text message parts

**Comment by yaelberger-commits** [comment]
Other piece of related content going to Content Critique this morning. Ensuring everything is consistent today. 
Steve M made progress
PR doesn't have to wait on content, can edit the templates later. Unblocks PR, ready for review some time today

**Comment by YedidaZalik** [comment]
This is merged. So dev side is done, waiting for final content.
Should go through QA steps and @sastels  will take on.

**Comment by YedidaZalik** [comment]
Has been released.

**Comment by amazingphilippe** [comment]
Blocked until we get the translations back

**Comment by YedidaZalik** [comment]
[Transation back](https://docs.google.com/document/d/18gnEkYKCnWv4YQCgvv5rSnZjdpA5KWyFCKh1HB0zxaM/edit#heading=h.gjdgxs) and ready for review by @amazingphilippe

**Comment by YedidaZalik** [comment]
@amazingphilippe reviewed

@smcmurtry to work on today

**Comment by sastels** [comment]
unblocked now!

**Comment by smcmurtry** [comment]
I have updated the email templates in staging with content from here: https://docs.google.com/document/d/18gnEkYKCnWv4YQCgvv5rSnZjdpA5KWyFCKh1HB0zxaM/edit#

I also updated the templates on Prod.

**Comment by YedidaZalik** [comment]
This needs QA as done in staging and prod
@jzbahrai to QA

**Comment by amazingphilippe** [comment]
Phil and Yedida will review templates in GC Notify for typos
